src/components/ActivitiesList/ActivitiesList.module.css

```diff
@@ -10,13 +10,22 @@
 }
 
 .list__h1 {
-  color: red;
+  color: blue;
 }
 
 .list__card {
-  list-style: none;
-  border: 2px solid red;
+  border: 2px solid blue;
   width: 400px;
   padding: 10px;
   margin: 10px;
 }
+
+.thumb_up {
+  background-color: limegreen;
```

Comment:
(I bet the icons from fontawesome have a way to set the colors of the icons themselves)